### PR TITLE
Fixed issue in XML for tiara tool

### PR DESCRIPTION
**Pull request by Minamehr**
Hi, this is for fixing an error in the Tiara tool's XML configuration.

### CODE REVIEW

File: tools/tiara/tiara.xml [hunk 1]
```diff
@@ -6,17 +6,12 @@
     <expand macro="biotools"/>
     <expand macro="requirements"/>
     <command detect_errors="exit_code"><![CDATA[ 
-        
+        pwd &&
```
Comment (bgruening) [inline tools/tiara/tiara.xml:9]:
```suggestion
```

File: tools/tiara/tiara.xml [hunk 2]
```diff
@@ -34,6 +29,17 @@
             --k2 $advanced_options.advance.second_stage_kmer
         #end if
 
+        #if $taxonomy_filter
+            --tf #for $tf in $taxonomy_filter
+                $tf
+            #end for
+        #end if
+        #if $taxonomy_filter
```
Comment (bgruening) [inline tools/tiara/tiara.xml:37]:
```suggestion
```

File: tools/tiara/tiara.xml [hunk 3]
```diff
@@ -34,6 +29,17 @@
             --k2 $advanced_options.advance.second_stage_kmer
         #end if
 
+        #if $taxonomy_filter
+            --tf #for $tf in $taxonomy_filter
```
Comment (bgruening) [inline tools/tiara/tiara.xml:33]:
```suggestion
            --tf
            #for $tf in $taxonomy_filter
```

File: tools/tiara/tiara.xml [hunk 4]
```diff
@@ -33,6 +28,15 @@
             --k1 $advanced_options.advance.first_stage_kmer
             --k2 $advanced_options.advance.second_stage_kmer
         #end if
+        #if $taxonomy_filter
+            --tf #for $tf in $taxonomy_filter
+                $tf
+            #end for
+            #for $tf in $taxonomy_filter
+              && ls -l ./results/
```
Comment (bgruening) [inline tools/tiara/tiara.xml:36]:
```suggestion
```

**Review by bgruening — COMMENTED** [review]
Mh, I guess it will fail you have not added the new test data or removed the old test data.

File: tools/tiara/tiara.xml [hunk 5]
```diff
@@ -107,7 +98,7 @@
                 <element name="arc" file="arc" ftype="fasta" />
                 <element name="bac" file="bac" ftype="fasta" />
                 <element name="euk" file="euk" ftype="fasta" />
-                <element name="main_result" file="main_result03.txt" ftype="txt" />
+                <element name="main_result" file="main_result02.txt" ftype="txt" />
```
Comment (bgruening) [inline tools/tiara/tiara.xml:101]: @Minamehr are those files really `txt`? They look to me like a `tabular`

File: tools/tiara/tiara.xml [hunk 6]
```diff
@@ -43,9 +47,6 @@
             <option value="bac">bacteria</option>
             <option value="arc">archea</option>
             <option value="euk">eukarya</option>
-            <option value="unk">unknown</option>
-            <option value="pro">prokarya</option>
```
Comment (bgruening) [inline tools/tiara/tiara.xml:47]: you are removing those?

File: tools/tiara/tiara.xml [hunk 7]
```diff
@@ -88,15 +88,6 @@
             <param name="taxonomy_filter" value="pla"/>
             <output_collection name="output" type="list">
                 <element name="main_result" file="main_result01.txt" ftype="txt"/>
-                <element name="pla" file="pla.fasta" ftype="fasta" />
-            </output_collection>
-        </test>
-        <test expect_num_outputs="1">
```
[the remark this answers is not in the document]
Comment (Minamehr) [reply tools/tiara/tiara.xml:93]: 1.they were testing same things just with two other inputs.

File: tools/tiara/tiara.xml (hunk 5 above)
Comment (Minamehr) [reply tools/tiara/tiara.xml:101]: This is what in the tool documentation mentioned about the main output: "the probabilities of belonging to individual classes are also written to out.txt". Additionally, in the tool example command, they used "tiara -i sample_input.fasta -o out.txt".  However, as you pointed out, the actual format of the file appears to be tabular with structured rows and columns.

**Pull request merged**